libraries/AP_HAL_ChibiOS/hwdef/2RAWH743/README.md

```diff
@@ -0,0 +1,97 @@
+# iFlight 2RAW H743 Flight Controller
+
+The iFlight 2RAW H743 is a flight controller produced by [iFlight](https://shop.iflight.com/Thunder-H7-Flight-Controller-Pro2200).
+
+## Features
+
+ - MCU - STM32H743 32-bit processor running at 480 MHz
+ - IMU - ICM42688P
+ - Barometer - SPL06-001
+ - OSD - AT7456E
+ - Onboard Flash: 1GBit (not useable by ArduPilot)
+ - 6x UARTs
+ - 9x PWM Outputs (8 Motor Output, 1 LED)
+ - Battery input voltage: 2S-6S
+ - BEC 5V 1.5A
+ - BEC 10V 1.5A
+
+## Pinout
+
+![iFlight 2RAW H743 Board Top](Thunder-H7-1.png "iFlight 2RAW H743 Top")
+![iFlight 2RAW H743 Board Bottom](Thunder-H7-2.png "iFlight 2RAW H743 Bottom")
+
+## UART Mapping
+
+The UARTs are marked Rn and Tn in the above pinouts. The Rn pin is the
+receive pin for UARTn. The Tn pin is the transmit pin for UARTn.
+
+ - SERIAL0 -> USB
+ - SERIAL1 -> UART1 (VTX HD, DMA-enabled)
+ - SERIAL2 -> UART2 (RX, DMA-enabled)
+ - SERIAL3 -> UART3
+ - SERIAL4 -> UART4 (GPS, DMA-enabled)
+ - SERIAL6 -> UART6 (ESC Telemetry)
+
+## RC Input
+
+RC input is configured on the R2 (UART2_RX) pin. It supports all serial RC
+protocols. For protocols requiring half-duplex serial to transmit
+telemetry (such as FPort) you should setup SERIAL3 as an RC input serial port,
+with half-duplex, pin-swap and inversion enabled.
+ 
+## FrSky Telemetry
+ 
+FrSky Telemetry is supported using the T3 pin (UART3 transmit). You need to set the following parameters to enable support for FrSky S.PORT
+ 
+  - SERIAL3_PROTOCOL 10
+  - SERIAL3_OPTIONS 7
+  
+## OSD Support
+
+The iFlight 2RAW H743 supports OSD using OSD_TYPE 1 (MAX7456 driver).
+
+## PWM Output
+
+The iFlight 2RAW H743 supports up to 9 PWM outputs. The pads for motor output
+M1 to M8 are provided in the motor connector, plus M9 on a separate pad for LED strip
+or another PWM output.
+
+The PWM is in 5 groups:
+
+ - PWM 1-2   in group1
+ - PWM 3-6   in group2
+ - PWM 7-8   in group3
+ - PWM 9     in group4
+
+Channels within the same group need to use the same output rate. If
+any channel in a group uses DShot then all channels in the group need
+to use DShot. Channels 1-8 support bi-directional dshot.
+
```

Comment:
add vtx pwr switch section

Reply:
Do you have an example @Hwurzburg ?

Reply:
https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_HAL_ChibiOS/hwdef/SpeedyBeeF405WING/Readme.md

Reply:
Thx, fixed